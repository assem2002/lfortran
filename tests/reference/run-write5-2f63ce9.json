{
    "basename": "run-write5-2f63ce9",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/write5.f90",
    "infile_hash": "5d5d604b32e128f6ff9c94fd138118f49b7df62bfb1350c28972bdf2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-write5-2f63ce9.stdout",
    "stdout_hash": "328e3d23b16bd6440b3fdded2595d1959bf988eb2efdc177e2d8eaea",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}