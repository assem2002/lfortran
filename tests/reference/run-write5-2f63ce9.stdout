hi @bye
first     , % @last      6 @C @Hello E @world!
